rllib/core/models/catalog.py

```diff
@@ -264,20 +258,47 @@ def get_action_dist_cls_dict(
 
         Args:
             action_space: Action space of the target gym env.
+            framework: The framework to use.
             deterministic: Whether to return a Deterministic distribution on input
                 logits instead of a stochastic distributions. For example for Discrete
                 spaces, the stochastic is a Categorical distribution with output logits,
                 while the deterministic distribution will be to output the argmax of
                 logits directly.
 
+
         Returns:
-                Mapping from framework to distribution class.
+            The distribution class for the given action space.
         """
-        distribution_dicts = {
-            "deterministic": {"torch": TorchDeterministic, "tf": TfDeterministic},
-            "gaussian": {"torch": TorchDiagGaussian, "tf": TfDiagGaussian},
-            "categorical": {"torch": TorchCategorical, "tf": TfCategorical},
-        }
+
+        if framework == "torch":
+            from ray.rllib.models.torch.torch_distributions import (
+                TorchCategorical,
+                TorchDeterministic,
+                TorchDiagGaussian,
+            )
+
+            distribution_dicts = {
```

Comment:
distribution_dict
-s

Reply:
it's ok :) if the tests pass I'll keep this around, if not I'll change it.